ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -511,6 +519,23 @@ class ServicesHomeScreen extends React.Component<Props, State> {
     );
   }
 
+  private renderLocalQuickSectionDeletion = (section: ServicesSectionState) => {
+    const onPressItem = () => {
+      if (this.props.userMetadata && this.props.selectedOrganizations) {
+        this.props.removeSelectedOrgnizationItem(
```

Comment:
If you compute here the new Set of selectedId you could reuse this function **saveSelectedOrganizationItems** instead of creating a new one (_*removeSelectedOrgnizationItem*_)


```typescript
const onPressItem = () => {
      if (this.props.userMetadata && this.props.selectedOrganizations) {
        const updatedAreasOfInterest = this.props.selectedOrganizations.filter(
          item => item !== section.organizationFiscalCode
        );
        const selectedItemIds = new Some(
          new Set<string>(updatedAreasOfInterest)
        );
        this.props.saveSelectedOrganizationItems(
          this.props.userMetadata,
          selectedItemIds
        );
      }
    };
```

Reply:
function removed within https://github.com/teamdigitale/io-app/pull/1330/commits/e26faa4b15772cae155c7ce79dc82468c67b2148